terragrunt/modules/aws-organization/groups.tf

```diff
@@ -111,6 +111,14 @@ locals {
         permissions : [aws_ssoadmin_permission_set.view_only_access] }
       ]
     },
+    # sync-team prod
+    {
+      account : aws_organizations_account.sync_team_prod,
+      groups : [
+        { group : aws_identitystore_group.infra-admins,
```

Comment:
I would like access to this as well if possible.

Reply:
If we're going to implemnent AWS sync (which I'd love to), having admin access to this account would mean escalating privileges to infra-admin levels. It's best to keep perms restricted here.

Reply:
I think we should grant read-only access (which is safe?) but that can happen separately.

Reply:
We would then need to restrict accessing the parameters from the read-only account, and make sure logs are sanitized.

Reply:
Sure. We should probably do that anyway for all humans (i.e., parameters shouldn't be readable by infra-admins either, IMO).